docs/variants/dell_optiplex/initial-deployment.md

```diff
@@ -76,7 +76,97 @@ page](../../dasharo-tools-suite/releases.md). Once you have obtained it, you can
 then proceed with following the [Dasharo zero-touch initial deployment
 section](../../dasharo-tools-suite/documentation.md#dasharo-zero-touch-initial-deployment)
 procedure. This will help you set up Dasharo effectively and without manual
-intervention.
+intervention. However, if you wish to flash firmware you have built yourself,
+you will need take an alternate route and go through firmware preparation.
+
+## Firmware preparation
+
+A fresh firmware binary built from source for this platform is not necessarily
+complete and ready to be deployed on hardware.
+
+If you have built it using the non-TXT config, you can safely go ahead and
+flash, the platform will be fully operational - but due to lack of EC firmware,
+the fans will always spin at full speed, which can be a nuisance.
+
+However, if you have build the firmware with TXT support, without patching the
+.rom with proprietary blobs the platform will simply refuse to boot.
+
+This brings us to patching the binary. This guide covers the process of
+obtaining 3 firmware blobs necessary for a full-feature deployment on the
+OptiPlex platform:
+
+* `sch5545_ecfw.bin` - EC firmware
+* `IVB_BIOSAC_PRODUCTION.bin` - Ivy Bridge BIOS ACM - **TXT support only**
+* `SNB_IVB_SINIT_20190708_PW.bin` - Ivy Bridge SINIT ACM - **TXT support only**
```

Comment:
I would use the names as they appear in CBFS, i.e. `txt_bios_acm.bin` and `txt_sinit_acm.bin`

Reply:
Fixed in [this commit](https://github.com/Dasharo/docs/pull/873/commits/2d83ab4937b0494d95d8012a011eb1fc4e69fefa)